homeassistant/components/recorder/statistics.py

```diff
@@ -94,6 +133,7 @@ def async_setup(hass: HomeAssistant) -> None:
     """Set up the history hooks."""
     hass.data[STATISTICS_BAKERY] = baked.bakery()
     hass.data[STATISTICS_META_BAKERY] = baked.bakery()
```

Comment:
Does it make sense to have a bakery for creating long term statistics? It stays in memory but we only use it once an hour ?

Reply:
It's also used for queries from frontend though, it probably makes sense to keep it in memory?